ONSdigital/design-system
# Fix: Add wide and full-width container classes to cookie banner options

### What is the context of this PR?
Fixes #2629 

### How to review
- Check cookie banner can be adjusted to match wide and fullwidth container layouts of other page elements.
- Check tests
